Warhammer 4e Character Sheet/sheet.json

```diff
@@ -2,7 +2,7 @@
 	"html": "Warhammer 4e Character Sheet.html",
 	"css": "Warhammer 4e Character Sheet.css",
 	"authors": "Djjus",
-	"roll20userid": "2472284, 3135709",
```

Comment:
This commit is removing the credit of the original sheet author Unless he has asked it to be remove please leave this here for now.

Thanks,
Cassie

Reply:
Ah, i was wondering who's it was. Fixed. Thanks Cassie.